third_party/tonic/dart_wrappable.cc

```diff
@@ -15,27 +15,48 @@ DartWrappable::~DartWrappable() {
   TONIC_CHECK(!dart_wrapper_);
 }
 
+// TODO(dnfield): Delete this. https://github.com/flutter/flutter/issues/50997
```

Comment:
Can we deprecate this so we ensure no one else starts using it?

Reply:
I don't think anyone is directly using it, and I was assuming that if I deprecate it now it would cause build errors.  I'm going to keep focusing on removing all references to it over the next week or so and just get it down.  It's not something we typically add more usages of.